jest-setup.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line import/no-unassigned-import
+require('@microsoft/jest-sarif');
```

Comment:
Since this is a JS repo, this works fine. Adding the side effect import is really only necessary for TS repos. I'll update the jest-sarif docs to reflect this.

Reply:
Ya, totally makes sense. If we observe issues in VSCode, we can move to imports in the file.